deploy/backplane/lpsre/rhoam/01-rhoam-lpsre-admins-project.ClusterRole.yaml

```diff
@@ -129,4 +129,31 @@ rules:
   resources:
   - deployments/scale
   verbs:
-  - patch
\ No newline at end of file
+  - patch
+# LP SRE can restart deployments/statefulsets/deploymentconfigs
+- apiGroups:
+  - apps
+  - extensions
+  resources:
+  - deployments
+  verbs:
+  - patch
+- apiGroups:
+  - apps
+  resources:
+  - statefulsets
+  verbs:
+  - patch
+- apiGroups:
+  - apps.openshift.io
+  resources:
+  - deploymentconfigs
```

Comment:
Does RHOAM use deploymentconfigs? Just want to explicitly confirm as the card doesn't mention it.

Reply:
Yes, updated MTSRE-965 with explicit references to the requiring SOP's